src/prefect/tasks/notifications/email_task.py

```diff
@@ -88,6 +96,7 @@ def run(
         msg_plain: str = None,
         email_to_cc: str = None,
         email_to_bcc: str = None,
+        attachments: List[str] = [],
```

Comment:
This should default to `None`. And we should add "attachments" to the `@defaults_from_attrs` decorator above.